dev/otel-config.yaml

```diff
@@ -32,11 +25,15 @@ exporters:
 service:
   pipelines:
     traces:
-      receivers: [otlp, jaeger, zipkin]
+      receivers: [otlp]
       processors: [batch]
       exporters: [jaeger, logging]
```

Comment:
Jaeger supports OTLP by default. We should be able to drop jaeger protocol here.

Probably separate PR. Ref: https://www.jaegertracing.io/docs/1.38/apis/#opentelemetry-protocol-stable